front/src/modules/activities/timeline/components/Timeline.tsx

```diff
@@ -95,6 +96,17 @@ export function Timeline({ entity }: { entity: CommentableEntity }) {
     },
   });
 
+  const StyledStartIcon = styled.div`
+    align-items: center;
+    align-self: stretch;
+    color: ${({ theme }) => theme.font.color.tertiary};
```

Comment:
const StyledStartIcon = styled.div`
    align-self: flex-start;
    color: ${({ theme }) => theme.font.color.tertiary};
    display: flex;
  `;

looks to be a more concise way to achieve the same goal!